### PR TITLE
Feature: make hero section prototype

### PR DESCRIPTION
**Pull request by teoh4770**
Issue: https://github.com/LaurierHawkHacks/Landing/issues/104
Branch: feature/104/make-hero-section-prototype

### What is this pr do?

Setup hero section with the amazing background ✨ 

### Concerns?

1. I have turn background svg into react component for animation purposes, but I'm not sure if this is normally done like this (we could apply animation class on svg component directly in this way).
2. Spacing issue, need a consistent spacing (padding around the section) across the page, not just individual one.

### Things to implement in the future

1. Animation on the little things 🚀 Any animation library suggestion?

### CODE REVIEW

**Comment by juancwu** [comment]
Can you resolves the conflicts?

**Comment by teoh4770** [comment]
let me try

File: src/assets/hero/mobile.svg
Comment (juancwu) [inline src/assets/hero/mobile.svg:1]: Why is this so tall? Was it cropped like this on purpose?

Comment (teoh4770) [reply src/assets/hero/mobile.svg:1]: yeah hhh, I also feel like it is quite tall

Comment (teoh4770) [reply src/assets/hero/mobile.svg:1]: how about a height of 500 for the mobile image, instead of 800? Would it be better @juancwu ?

Comment (teoh4770) [reply src/assets/hero/mobile.svg:1]: let me try to fix it

**Review by GabeDiniz — COMMENTED** [review]
It looks like the Navbar is cutting off the top of the SVG
image

Not sure about this one but: in mobile view, the title/date looks odd. Might be easier to keep the date on the SVG (although it could appear too small there). 
image

**Comment by teoh4770** [comment]
We can discuss it at tomorrow meeting @GabeDiniz

**Comment by juancwu** [comment]
image

@teoh4770 Here are the updates for the hero section. Can you add the new things in this PR as well?

Basically, is a subtitle and everything has been moved up a little closer to the Navbar.

Feel free to take a look in Figma.

**Comment by teoh4770** [comment]
I think it is ok now, can you check?

**Comment by teoh4770** [comment]
Besides the font size is bad imo, the layout is ok for now.

**Review by GabeDiniz — APPROVED** [review]

**Review by juancwu — APPROVED** [review]
Minor thing we need to fix later is to keep the font sizes as close as possible to the figma design. The HawkHacks title and the subtitle now don't have big enough of a difference that makes it look weird.

**Pull request merged**